.github/workflows/installers.yml

```diff
@@ -79,4 +92,136 @@ jobs:
             staging: ${{ inputs.staging }}
             architecture: ARM64
             operating_system: self-hosted
-        secrets: inherit
\ No newline at end of file
+        secrets: inherit
+
+    # Run and update nightly release when the nightly input is set to true or if the schedule is triggered
+    update-nightly-release:
+        if: ${{ inputs.nightly || github.event_name == 'schedule' }}
+        runs-on: ubuntu-latest
+        needs: [ windows-installer, linux-modern-installer, linux-legacy-installer, macos-installer, macos-arm-installer ]
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Download the run info
+                uses: actions/download-artifact@v2
+                with:
+                    name: macos-run-info
+
+            -   name: Set the run info as environment variables
+                run: |
+                    . run_info.sh 
```

Comment:
Last little remark: Did you mean to write `./run_info.sh` here? It works this way as well, so I'm not changing it, just a question.

Reply:
No, the `. <file>` with the space after the dot is basically `source <file>` which reads and executes the content. But `source` is not always present in all Unix systems so the `.` is the safer more portable choice

But the space is there intentional